app/src/main/java/org/mozilla/fenix/settings/logins/SavedLoginsAuthFragment.kt

```diff
@@ -256,8 +257,9 @@ class SavedLoginsAuthFragment : PreferenceFragmentCompat(), AccountObserver {
         }
     }
 
+    @Suppress("Deprecation")
     private fun showPinVerification() {
-        val manager = activity?.getSystemService(KEYGUARD_SERVICE) as KeyguardManager
+        val manager = activity?.getSystemService<KeyguardManager>()!!
```

Comment:
I don't think we should use un-safe calls if the newer target SDK asserts them as nullable. Some devices maybe not support some system services and we shouldn't crash because of that.

Reply:
We're already using unsafe calls as we case nullable to not-null. The SDK doesn't assert if a system service is null or not, the extension method simply always returns a nullable type.